Doc/library/asyncio-graph.rst

```diff
@@ -0,0 +1,131 @@
+.. currentmodule:: asyncio
+
+
+.. _asyncio-graph:
+
+===================
+Stack Introspection
+===================
+
+**Source code:** :source:`Lib/asyncio/graph.py`
+
+-------------------------------------
+
+asyncio has powerful runtime call graph introspection utilities
+to trace the entire call graph of a running coroutine or task, or
+a suspended *future*.  These utilities and the underlying machinery
+can be used by users in their Python code or by external profilers
+and debuggers.
+
+.. versionadded:: 3.14
```

Comment:
```suggestion
.. versionadded:: next
```

Reply:
It's for the next release version not for 3.15. That's a new sphinx extension that we use. IOE it will render (3.14.0a2, unreleased).

Reply:
wow, that's great. Applying.